webapp/app.py

```diff
@@ -78,3 +81,9 @@ def configure_error_handlers(app: App) -> None:
     @app.errorhandler(Exception)
     def handle_exception(error: Exception):
         return error_dispatcher.dispatch_error(error, request)
+
+
+@celery.on_after_configure.connect
+def configure_periodic_tasks(app, **kwargs):
```

Comment:
`app` is never used here, but I guess that it's needed because it will be always passed, and we would get an error if we didn't have is as parameter?

Reply:
Oh, fragment from my experiments ending up changing the config helper. Fixed.

Reply:
Is `kwargs` needed though?

Reply:
Yes, signal recipients (that's the decorator) always need **kwargs in their functions